A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/MFD/Images/A32NX_DECEL.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 64 64" style="enable-background:new 0 0 64 64;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#ED008C;}
```

Comment:
Please use the shared color vars from A32NX_Display_Common.css. Also, I think you should be able to use CSS selectors from A320_Neo_MFD.css, so you wouldn't have to declare it in the SVG

Reply:
Would you mind providing an example?  i'm not a CSS wizz :)  thanks for pointing this out.

Reply:
Oh, I'm also not a CSS pro. Now that I think about it, maybe it actually isn't possible, since I think it's taking the filepath it may be taking it as a link. I don't know if it's possible then. Can you include css files in SVG? I don't think so

Reply:
ill do some research and look into it to see if it can be done this way.  if not ill correct the color to the correct one regardless.

Reply:
i think this would be possible if the svg were inlined, but not if its seperate.